common/src/components/ScrollSpyMenu/index.tsx

```diff
@@ -2,12 +2,38 @@ import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import Scrollspy from 'react-scrollspy';
 import AnchorLink from 'react-anchor-link-smooth-scroll';
+import { detect } from 'detect-browser';
 import { DrawerContext } from '../../contexts/DrawerContext';
 import Button from 'reusecore/src/elements/Button';
 import { Link } from 'gatsby';
 
 import { HoverMenu } from './scrollSpyMenu.style';
 
+const onClickDownload = (e: Event) => {
+  e.preventDefault();
+  const browser = detect();
+  let downloadLink = '';
+  switch (browser && browser.name) {
+    case 'chrome':
+      downloadLink = 'https://chrome.google.com/webstore/detail/abkfbakhjpmblaafnpgjppbmioombali';
+      break;
+    case 'firefox':
+      downloadLink = 'https://addons.mozilla.org/en-US/firefox/addon/worldbrain/';
+      break;
+    case 'brave':
+      downloadLink = 'https://chrome.google.com/webstore/detail/abkfbakhjpmblaafnpgjppbmioombali';
+      break;
+    default:
+      // TODO: Fallback case? Default is Chrome link
```

Comment:
@blackforestboi What should we do when our extension does not support user's browser?
Redirect to comming soon support/request support is what I'm thinking, we need design if choose this solution.

Reply:
very good point. Maybe we can show a little popup: "Your browser is not supported" + 1 image
If you implement the functionality, I design the popup itself.